tests/testthat/_snaps/expect-match.md

```diff
@@ -5,23 +5,39 @@
 ---
 
     `one` does not match regexp "asdf".
-    Text: "bcde"
+    Text:
+    ✖ │ bcde
```

Comment:
Will the UTF-8 snapshots cause issues on non-UTF-8 platforms? Maybe testthat does something about this, I forgot.